plugins/steve/resource-instance.js

```diff
@@ -93,6 +94,15 @@ export default {
     return this.id || Math.random();
   },
 
+  columns() {
+    return this.schema?.attributes?.columns?.reduce((agg, attribute) => {
+      return {
+        ...agg,
+        [attribute.name.toLowerCase()]: get(this, attribute.field)
+      };
+    }, {});
+  },
+
```

Comment:
`attributes.columns` is the info k8s returns when asked for a "table view" of the resource i.e. what's shown in `kubectl get $type -o wide`.  We use this for the generic `list` pages, but you shouldn't write code that relies on a particular column being there or having a particular name or value because they could vary in different k8s releases or change in any future one.

Grab the value you need from the appropriate part of the spec/status (internal/externalIp come from `status.addresses`).

Reply:
Removed.